api/exclusions/exclusionService.ts

```diff
@@ -0,0 +1,22 @@
+import { NextFunction, Response } from 'express';
+import { EnhancedRequest } from '../lib/models';
+
+export async function getUserExclusions(req: EnhancedRequest, res: Response, next: NextFunction) {
+    const isJudge = req.session.passport.user.userinfo.roles.includes('caseworker-ia-iacjudge');
+    let exclusions;
```

Comment:
We could just initialise the exclusions array as `const exclusions = []`.  if `isJudge` is truthy push the exclusion to `Exclusion[] `. No need for the else statement

Reply:
This is a mock code that is a temporary code. But, I agree with your comment in principle